src/content/en/tools/workbox/v3/modules/workbox-cacheable-response.md

```diff
@@ -0,0 +1,148 @@
+project_path: /web/tools/workbox/v3/_project.yaml
+book_path: /web/tools/workbox/v3/_book.yaml
+description: The module guide for workbox-cacheable-response.
+
+{# wf_updated_on: 2017-11-27 #}
+{# wf_published_on: 2017-11-27 #}
+
+# Workbox Cachable Response  {: .page-title }
```

Comment:
The title of this entry is Workbox Cachable Response but the description uses 'Cacheable'. Typo for the earlier one?